### DISCUSSION TITLE
Should I use traefik + FPM-NGINX variation  ?

### DISCUSSION THREAD

**Discussion by IgordeOliveira** [serversideup/docker-php#264] [Q&A] [answered]
I'm not sure if I missed something in the documentation or if my knowledge about Reverse Proxies/SSL is insufficient, but since the image contains an nginx, should I use a traefik pointing to the docker or is it enough to only expose the 80 and 443 ports from the container with nginx? My goal is to run a Laravel application with SSL for a subdomain in a Linux VM.

**Comment by jaydrogers** [comment, marked as answer]
Yup! We use FPM + NGINX to serve our apps with Traefik sitting in front.

Here is an example app (generated from our other project Spin): https://github.com/serversideup/spin-laravel-demo-app

Learn more about how Spin works here: https://twitter.com/jaydrogers/status/1740118184970101197

More on the project and what we're working on is here: https://github.com/serversideup/spin/pull/24
